src/components/Drawer.js

```diff
@@ -268,14 +273,14 @@ class Drawer extends React.Component {
                     />
                   }
                 </span>
-                <h1 style={styles.title}>
+                <h1 id='mx-drawer-title' style={styles.title}>
```

Comment:
I think the `id` here is too general. If there are multiple `Drawer` components open won't the `id`s conflict?

You could use lodash's `uniqueId` to make the id more unique.

Reply:
yep, good idea.  I'll update